browser-test/src/admin/admin_publish.test.ts

```diff
@@ -70,6 +70,63 @@ test.describe(
   },
 )
 
+test.describe(
+  'publishing all programs with disabled programs feature flag on',
+  {tag: ['@uses-fixtures']},
+  () => {
+    test('shows programs and questions that will be publised in the modal, including disabled programs', async ({
+      page,
+      adminPrograms,
+      adminQuestions,
+    }) => {
+      const disabledProgram = 'Disabled test program'
+      const publicProgram = 'Public test program'
+      const questionName = 'admin-publish-test-address-q'
+      const questionText = 'admin-publish-test-address-q'
+      // CreateNewVersion implicitly updates the question text to be suffixed with " new version".
```

Comment:
**nitpick:** Could we make this the actual function name? `adminPrograms.createNewVersion` - you also may be able to use the TS @link here, but not sure if it works in a comment like this.